utils.ts

```diff
@@ -833,3 +837,78 @@ export async function updateDbFromEthTransaction(txId: string) {
 
   console.log("Processed logs for tx", txId);
 }
+
+ export async function storeOffchainAttestation(pkg: AttestationShareablePackageObject) { 
+  console.log("Storing offchain attestation", pkg);
+
+  const config: OffchainConfig = {
+    address: pkg.sig.domain.verifyingContract,
+    version: pkg.sig.domain.version,
+    chainId: BigInt(pkg.sig.domain.chainId)
+  };
+
+  const offchain = new Offchain(config, pkg.sig.message.version, new EAS(ZERO_ADDRESS));  
+  const isValidAttestation = offchain.verifyOffchainAttestationSignature(pkg.signer, pkg.sig);
```

Comment:
`verifyOffchainAttestationSignature` checks the format of the offchain attestation.